tools/mkcomposefs.c

```diff
@@ -325,8 +325,9 @@ static void usage(const char *argv0)
 		"  --user-xattrs         Only store user.* xattrs\n"
 		"  --print-digest        Print the digest of the image\n"
 		"  --print-digest-only   Print the digest of the image, don't write image\n"
-		"  --from-file           The source is a dump file, not a directory\n",
-		bin);
+		"  --from-file           The source is a dump file, not a directory\n"
+		"  --format-version=N    Use this format version (default=%d)\n",
+		bin, LCFS_VERSION_MAX);
```

Comment:
There's going to be an interesting question going forward of how quickly we bump the default version.  Callers should probably pin it...but they couldn't do that *before* this change because we didn't expose the option.